docs/opening-with-cps.md

```diff
@@ -0,0 +1,35 @@
+# What determines when a project is opened with the new project system?
+
+When opening a solution, there are guids stored in the .sln file that tell VS to find a project system registered with that Guid. For example, if the sln file has an entry like 
+
+```
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ClassLibrary3", "ClassLibrary3\ClassLibrary3.csproj", "{5B9CCABF-2F7E-407C-ACD7-53C37BB859EE}"
+EndProject
+```
+
+VS first looks for a type called IVsProjectSelector that's registered with that guid. If there is one, it asks the selector to give it the guid of the project system that should be used to open the project.
+If there is no selector, then that guid is the one that's used.
+
+In the case above, `FAE04EC0-301F-11D3-BF4B-00C04F79EFBC` is the guid of the old project system. However, there is also a IVsProjectSelector registered for that guid. That project selector sniffs the project to look for a `TargetFramework` or `TargetFrameworks` property and if one exists open the project with the new project system. Otherwise it falls back to the old project system.
```

Comment:
> That project selector sniffs the project to look for a `TargetFramework` or `TargetFrameworks` property

Consider linking to the issue noting that it only looks in the direct project file for the attribute.  Not in any included file.